### DISCUSSION TITLE
ANNOUNCEMENT: Lifting Index Limitations for Community License

### DISCUSSION THREAD

**Discussion by DejanMilicic** [ravendb/ravendb#18564] [Announcements]
We are excited to announce an important update for our Community license users. Effective immediately, we are lifting the restrictions on the number of static indexes.

The Community license, designed for small and hobby projects, originally included generous limits to support project growth and development. Recently, we introduced some limitations, including a cap of 12 static and 24 auto indexes per database. We quickly realized that this specific restriction could hinder project growth, as it's difficult to predict the number of indexes a project might need in its early stages.

After listening to feedback from our community, we understood how this limitation impacted our users. In response to your concerns and the community's input, we have removed this limitation entirely. From today onwards, the number of static and auto indexes for all newly issued Community licenses will no longer be limited.

What action is needed on your side for existing installations? If your cluster can reach `api.ravendb.net`, your Community license has already been updated. You can verify this by opening `/license/status` endpoint and checking the following keys and their values:

```
"MaxNumberOfStaticIndexesPerDatabase": null
"MaxNumberOfStaticIndexesPerCluster": null
"MaxNumberOfAutoIndexesPerDatabase": null
"MaxNumberOfAutoIndexesPerCluster": null
```

If these keys were not updated, the connection to `api.ravendb.net` failed repeatedly. You can manually force a license update by opening `/studio/index.html#about` endpoint and clicking on `Force Update`. If this manual license refresh action fails as well, you will have to request a new Community license at https://ravendb.net/license/request/community.

We appreciate your feedback and are committed to making RavenDB as flexible and supportive as possible for all your projects.

Thank you for being a part of our community!

**Comment by ashokgudur** [comment]
Wow Super!

Thanks RavenDB Team. ❤️
